monarchmoney/monarchmoney.py

```diff
@@ -2377,6 +2457,71 @@ async def upload_account_balance_history(
             if resp.status != 200:
                 raise RequestFailedException(f"HTTP Code {resp.status}: {resp.reason}")
 
+            response = await resp.json()
+            session_key = response["session_key"]
+            return session_key
+
+    async def parse_upload_balance_history_session(self, session_key: str) -> dict:
+        """
+        Triggers parsing of the uploaded balance history CSV file.
+
+        :param session_key: The session key for the uploaded file.
+        """
+
+        query = gql(
+            """
+            mutation Web_ParseUploadBalanceHistorySession($input: ParseBalanceHistoryInput!) {
+                parseBalanceHistory(input: $input) {
+                    uploadBalanceHistorySession {
+                        ...UploadBalanceHistorySessionFields
+                        __typename
+                    }
+                    __typename
+                }
+            }
+            fragment UploadBalanceHistorySessionFields on UploadBalanceHistorySession {
+                sessionKey
+                status
+                __typename
+            }
+            """
+        )
+
+        variables = {"input": {"sessionKey": session_key}}
+
+        return await self.gql_call(
+            "Web_ParseUploadBalanceHistorySession", query, variables
+        )
+
+    async def get_upload_balance_history_session(self, session_key: str):
```

Comment:
for consistency with similar methods in this API 

```suggestion
    async def is_upload_balance_history_complete(self, session_key: str) -> bool:
```

Reply:
done!